README.md

```diff
@@ -1,2 +1,12 @@
 # aspose-imaging-probe
 This repository is for relevant probation tasks
+
+## Keywords
+
+edit a photo, editing of pictures, edit an image, photo background editor, photo background change, resizing an image, resizing photos, photo sizing, picture resizer, image crop, crop photo, filter photo, online photo editor, online image editor, photo editing app, free photo editor
```

Comment:
1

Reply:
Yes.

Reply:
@anms23 , good, now I'm able to write review comments, thanks for answer.

Reply:
@anms23 , is photo background editor relevant keyword for presented article? Is top10 keyplanner phrases presented in your report?

Reply:
I've checked the keywords and fixed their usage.
https://github.com/aspose-imaging/aspose-imaging-probe/commit/87620b4e18177aa9a3f6a3c985a2e582a73a0f58